fl4health/strategies/feddg_ga.py

```diff
@@ -290,10 +334,9 @@ def aggregate_evaluate(
         self.evaluation_metrics = {}
         for client_proxy, eval_res in results:
             cid = client_proxy.cid
+            # make sure that the metrics has the desired loss key
+            assert FairnessMetricType.LOSS.value in eval_res.metrics
             self.evaluation_metrics[cid] = eval_res.metrics
-            # adding the loss to the metrics
-            val_loss_key = FairnessMetricType.LOSS.value
-            self.evaluation_metrics[cid][val_loss_key] = eval_res.loss
```

Comment:
No need to force the loss into the dictionary here. It should be packed with the metrics. If it's not, something else has gone wrong.